src/gentropy/datasource/eqtl_catalogue/study_index.py

```diff
@@ -59,37 +60,34 @@ class EqtlCatalogueStudyIndex:
     def _identify_study_type(
```

Comment:
You are right that cols here don't need to be parametrised strictly speaking. Now the approach is hybrid: one is a parameter, the other one is not. I suggest following one or the other